data/language/dutch.txt

```diff
@@ -2382,19 +2382,19 @@ STR_2380    :{SMALLFONT}{BLACK}Kleiner stuk water aanpassen
 STR_2381    :{SMALLFONT}{BLACK}Groter stuk water aanpassen
 STR_2382    :Land
 STR_2383    :Water
-STR_2384    :{WINDOW_COLOUR_2}Je doel:
+STR_2384    :{WINDOW_COLOUR_2}Je doelstelling:
```

Comment:
Wiktionary lijkt hetzelfde te suggeren als FWest98. Dan blijft er nog één criterium over: is het duidelijk genoeg voor een basisschoolleerling (groep 7/8)? De context zal wel helpen denk ik.